core/src/main/java/com/expediagroup/streamplatform/streamregistry/core/validators/StreamValidator.java

```diff
@@ -57,11 +58,15 @@ private void requireExistingDomain(Stream stream) {
   }
 
   private void requireExistingSchema(Stream stream) {
-    if (stream.getSchemaKey() == null) {
-      throw new ValidationException("Schema must be specified");
-    }
+    requireSchemaSpecified(stream);
     if (!schemaView.exists(stream.getSchemaKey())) {
       throw new ValidationException("Schema does not exist");
     }
   }
+
+  private void requireSchemaSpecified(Stream stream) {
```

Comment:
Doing this validation from egsp-stream-reg for update case.
`!schemaView.exists(stream.getSchemaKey())`